src/oracle11_fmt_plug.c

```diff
@@ -165,20 +171,52 @@ static void done(void)
 
 static int valid(char *ciphertext, struct fmt_main *self)
 {
-	int extra;
-	return hexlenu(ciphertext, &extra)==CIPHERTEXT_LENGTH && !extra;
+	char *p = ciphertext;
+
+	if (!memcmp(ciphertext, FORMAT_TAG, FORMAT_TAG_LENGTH))
+		p += FORMAT_TAG_LENGTH;
+
+	if (strlen(p) != CIPHERTEXT_LENGTH)
+		return 0;
+
+	if (!ishex(p))
+		return 0;
```

Comment:
I think you changed the format from only accepting uppercase hex, to any case. Was this intentional? I'm not sure what format you usually get when doing whatever one does to fetch oracle hashes, so I can't tell - it might well be sensible.

Anyway if we do this I think we should unify case in split as well.

Reply:
You're absolutely right.

I actually did it intentionnally. I was extracting the hashes directly from the output of `hexdump -C` applied to an ORA password file, and the hex output is lower case there. My local test file contains a hash in upper case, and one in lower case as well. I was wondering if this change had any implication whatsoever.

It seems this has been changed in commit 81391c139d (fixed split-case bugs.  Some for #1742).

I'm going add a new commit to this Pull Request to either revert this change back, or unify case in a custom split function.
I'm going to have a look at this as soon as i can. Thanks for your feedback.

Reply:
You should also add `FMT_SPLIT_UNIFIES_CASE` format flag.